test/functional/eval/buf_functions_spec.lua

```diff
@@ -279,9 +279,9 @@ describe('setbufvar() function', function()
     eq(false, winmeths.get_option(windows[3], 'number'))
     eq(false, winmeths.get_option(meths.get_current_win(), 'number'))
 
-    eq(false, meths.get_option('hidden'))
-    funcs.setbufvar(1, '&hidden', true)
     eq(true, meths.get_option('hidden'))
+    funcs.setbufvar(1, '&hidden', 0)
```

Comment:
is `false` intentionally not used here?
```suggestion
    funcs.setbufvar(1, '&hidden', false)
```

Reply:
Yes, because in Lua `0` is not `false`, so it doesn't translate correctly: it throws an error that `'hidden'` must be a number.